edgedb-client/src/builder.rs

```diff
@@ -60,13 +60,14 @@ pub struct Builder {
     database: String,
     pem: Option<String>,
     cert: rustls::RootCertStore,
-    verify_hostname: Option<bool>,
+    client_security: TlsSecurity,
```

Comment:
Should we also rename `client_security` to `tls_security`?

Reply:
Yes, sure. This is private var, so doesn't matter much.